### DISCUSSION TITLE
梅林+阿里云WebDAV+Infuse 是不是又不行啦？

### DISCUSSION THREAD

**Discussion by FFFoursix** [messense/aliyundrive-webdav#737] [Q&A] [answered]
有没有小伙伴是梅林 + 阿里云WebDAV + Infuse，目前 Infuse 能读取出影片，但是播放时提示「加载此内容发生错误」。用的是两周前的v1.11.0版，还有办法可以解决吗TAT？

**Comment by KimiMeta** [comment]
梅林+阿里云WebDAV+Infuse ，我目前也无法使用！

**Comment by gejun9558** [comment]
软路由+阿里云WebDAV+Infuse ，我也无法使用！

**Reply by gejun9558** [reply]
已换成alist 阿里云盘open，目前可使用

**Reply by FFFoursix** [reply]
求教～alist+阿里云盘open，要怎么配置呀？

**Reply by Eri5Ch** [reply]
仔细看alist的文档 也需要折腾一番

**Reply by FFFoursix** [reply]
> 仔细看alist的文档 也需要折腾一番

是不是需要买个软路由呀？

**Reply by Eri5Ch** [reply]
alist在梅林软件库里面就有

**Reply by lyonfoxw** [reply]
梅林软件中心有alist吗

**Reply by FFFoursix** [reply]
> 梅林软件中心有alist吗

有的，直接搜索就行。可惜我的空间不够了...装不了。

**Comment by lyonfoxw** [comment]
情况一模一样

**Comment by quenqq** [comment]
是的，只能用alist了

**Comment by Nemolovesclaire** [comment, marked as answer]
刚看到Ｖ2.0.1 里“移除了 Koolshare 梅林固件路由器平台支持”， 大神是不会再给梅林出这个更新了？

**Reply by FFFoursix** [reply]
大神在其他帖子回复了，不会再更新了，之后没了。
